jenkins-builds/7.0.UP02-20.11-dom-mc

```diff
@@ -27,6 +27,7 @@
  IDL-8.5.1.eb                                           --set-default-module
  IDL-8.7.2-CSCS.eb
  ipcmagic-0.1-CrayGNU-20.11.eb                          --set-default-module
+ ipcmagic-1.0.1-CrayGNU-20.11.eb 
```

Comment:
```suggestion
 ipcmagic-1.0.1-CrayGNU-20.11.eb                       --set-default-module
```

Reply:
Thanks @twrobinson: I suggest to remove the old release on Dom, so we can move forward with `UpdateEB`.